src/xdist/workermanage.py

```diff
@@ -220,8 +222,7 @@ def __init__(self, nodemanager, gateway, config, putevent):
         self.workerinput = {
             "workerid": gateway.id,
             "workercount": len(nodemanager.specs),
-            "slaveid": gateway.id,
```

Comment:
I've removed these as they are not used anywhere. They must be old remnants.

Reply:
removal is a breaking change for all consumers and would require a major release

Reply:
I don't see it being exposed anywhere and it's also not even documented anywhere. I'm happy to leave it in but I don't see how anyone could have used this but probably I'm missing something.

Reply:
pytest cov uses this

https://github.com/pytest-dev/pytest-cov/commit/b33126da4f4a2d4b52f90564ba0760a7668fc837

Reply:
Good point! I didn't check for the getattr stuff. I've reverted this part of the changes. Wasn't really related anyway. :)